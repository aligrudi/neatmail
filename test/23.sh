# neatmail test
./mail pn -H -s0 -b$1 <$1 >/tmp/.nmtest.box 2>/dev/null
./mail pg -s /tmp/.nmtest.box 1
rm /tmp/.nmtest.box
