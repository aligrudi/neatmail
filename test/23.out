From root Tue Dec 22 02:02:02 2015
From: B <b@b.eu>
To: A <a@a.eu>
Subject: Message from B
Message-Id: <msg002>
Date: Tue, 22 Dec 2015 02:02:00 +0100

This is B.

